tests/integration/utils.py

```diff
@@ -60,10 +60,12 @@
     "https://raw.githubusercontent.com/CycloneDX/specification/1.4/schema/bom-1.4.schema.json"
 )
 
+DEFAULT_OUTPUT_DIR = Path("cachi2-output")
```

Comment:
You should pull this one from `cachi2/interface/cli.py` so that we won't have to change it at 2 places soon ;)